index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo-nav">
+            <div class="logo">
+            <h1>My Logo</h1>
+            </div>
+            <div class="nav-bar">
+                <ul>
+                    <li><a href="#portfolio-1">portfolio</a></li>
+                    <li><a href="#about-1">about</a></li>
+                    <li><a href="#contact-1">contact</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="container-1">
+            <div class="discription">
+                <h1>I'm Girma Glad to see you!</h1>
+                <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! if you like what you  see and have a project you need coded, don't hestiate to contact me</p>
+                <div class="let-s">
+                    <h1>LET'S CONNECT</h1>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="https://github.com/girma336" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="main-part" id="portfolio-1">
+        <article>
+            <div class="back-image-file all-image-pro"></div>
+            <div class="back-description">
+                <h1>Tonic</h1>
+                <div>
```

Comment:
- [ ] Kindly replace `h1` with `h2` to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) due to SEO purpose :blush: and one html page contains only `h1` which you already have at the homepage of your portfolio and all the other headings are between `h2` - `h6` w.r.t [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :thinking:

Reply:
Fixed.